iset.mm

```diff
@@ -67402,6 +67432,36 @@ readily usable (e.g., by ~ djudom and ~ djufun ) while the simpler
       AIWJTVGYSWKYADWTWQWLWMAXCBDAYCBSQXCBPUMYSSYABIWNWMWOWP $.
   $}
 
+  ${
+    caseinr.f $e |- ( ph -> Fun F ) $.
+    caseinr.g $e |- ( ph -> G Fn B ) $.
+    caseinr.a $e |- ( ph -> A e. B ) $.
+    $( Applying the "case" construction to a right injection.  (Contributed by
+       Jim Kingdon, 12-Jul-2023.) $)
+    caseinr $p |- ( ph -> ( case ( F , G ) ` ( inr ` A ) ) = ( G ` A ) ) $=
```

Comment:
As usual, my inclination would have been to state it with `` ( ( inr |` B ) ` A ) `` since A is assumed to be in B and G is a function on  B, but I haven't checked how the proof and typical usages would be affected. Up to you.

Reply:
There are actually some parts of this pull request where I do use some of the theorems which restrict `inr` and `inl`. So at least in that sense our practice is more similar than it might appear.

But you are also right that I don't do this everywhere. And "too cumbersome" is a decent summary of the reason.